docs/src/contributing.md

```diff
@@ -0,0 +1,67 @@
+# Contributing to Javis
+
```

Comment:
Having this file as contributing.md is helpful such that GitHub refers to this document when someone opens a PR. I'm not sure whether it's helpful to have it as part of the documentation.

Reply:
@TheCedarPrince suggested including it with the README.md. I think having it would be helpful but it won't be harmful to have as part of the documentation page.

Reply:
Oh I see. I thought you moved it but you copied it over, right? I think it makes sense we just need to be careful that we change both if we update the guidelines. 
Or maybe we can just use the contributing.md and build it via Documenter. Not sure whether or needs to be inside the `docs/src/` folder for that.